04-Multi-Band-Raster.Rmd

```diff
@@ -223,16 +220,29 @@ RGB_band2
 ```
 
 Notice that band 2 is band 2 of 3 bands.
+`band: 2  (of  3  bands)`
 
-`band        : 2  (of  3  bands)`
 
-#Raster Stacks in R
+#Challenge: Making sense of single band images
+If you compare the plots of band 1 (red) and band 2 (green) you can see subtle 
+differences in the image. The forest appears paler in band 2 (green). Why?  
+
 
-We have now explored several bands in a 3 band raster. Our data is a color image 
-which means it has atleast 3 bands. Let's bring in all of the bands and composite 
-it to create a final color RGB plot. To bring in all bands of a raster, we will use
-the `stack` function.
 
+Recall from our creation of the greyscale palette that greyscale is actually a 
```

Comment:
This me being confused as to why band 2 was a paler image than band 3 and then reasoning through it.  Please review closely as I may be way off.